src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/UpdateDocumentById.cs

```diff
@@ -23,6 +25,47 @@ NpgsqlTransaction transaction
 public class UpdateDocumentById(ISqlAction _sqlAction, ILogger<UpdateDocumentById> _logger)
     : IUpdateDocumentById
 {
+
+    /// <summary>
+    /// Returns the ReferentialId Guids and corresponding partition keys for all of the document
+    /// references in the UpdateRequest.
+    /// </summary>
+    public static DocumentReferenceIds DocumentReferenceIdsFrom(IUpdateRequest updateRequest)
```

Comment:
I'm sure you already plan this, but a reminder to extract these from here and my `UpsertDocument.cs` somewhere, probably a static class. I'm not a huge fan of using inheritance to share methods that can be static.